ui/perfherder/compare/CompareSelectorView.jsx

```diff
@@ -130,18 +136,24 @@ export default class CompareSelectorView extends React.Component {
                 )}
               </Col>
             </Row>
+            <Row className="justify-content-center">
+              {showWarning && (
+                <Alert color="warning">
+                  It is not recommended to compare a <b>try</b> build against a{' '}
+                  <b>mozilla-central</b> build.
```

Comment:
This is a nice improvement. Is there a link where someone could find out more?  I ask because I'm one of those people who likes to know 'why', but perhaps this is more obvious for Firefox engineers :)

Reply:
I think we could inline the explanation by extending this message with ", unless it is based on latest mozilla-central.".